packages/mdc-select/README.md

```diff
@@ -276,13 +244,12 @@ these mixins within CSS selectors like `.foo-select` to apply styling.
 
 Mixin | Description
 --- | ---
-`mdc-select-ink-color($color)` | Customizes the text entered into the select. This style applies only to the css version.  
+`mdc-select-ink-color($color)` | Customizes the color of the selected item displayed in the select. On css version, this also customized the color of the label.
 `mdc-select-container-fill-color($color)` | Customizes the background color of the select. 
 `mdc-select-label-color($color)` | Customizes the label color of the select in the unfocused state. 
```

Comment:
Add that this only covers the JS only version....otherwise label-color is covered by ink-color for CSS only